terraform/modules/spack/gitlab_ses.tf

```diff
@@ -0,0 +1,42 @@
+locals {
+  gitlab_email_domain = "gitlab.${var.ses_email_domain}"
+  smtp_secret_name = "gitlab-ses-secrets"
+  smtp_secret_password_key = "smtp-password"
+}
+
+resource "kubectl_manifest" "ses_config_map" {
+  yaml_body = <<-YAML
+    apiVersion: v1
+    kind: ConfigMap
+    metadata:
+      name: gitlab-ses-config
+      namespace: gitlab
+    data:
+      values.yaml: |
+        global:
+          email:
+            from: admin@${local.gitlab_email_domain}
+            reply_to: noreplay@${local.gitlab_email_domain}
```

Comment:
Is this a typo?
```suggestion
            reply_to: noreply@${local.gitlab_email_domain}
```

Reply:
This typo wasn't introduced by this PR, so I'll address this in a follow up